src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/editors/media.controller.js

```diff
@@ -49,6 +49,11 @@ angular.module("umbraco")
                     }
                 }
 
+                // set default size if no crop present (moved from the view)
+                if (url.indexOf('?') == -1)
+                {
+                    url += "?width=800&upscale=false"
```

Comment:
We need to add &animationprocessmode=false to this one too for http://issues.umbraco.org/issue/U4-7720